addons/web/static/src/js/core/dialog.js

```diff
@@ -309,6 +309,7 @@ var Dialog = Widget.extend({
             });
             $button.on('click', function (e) {
                 var def;
+                $button.find('button').attr('disabled', true);
```

Comment:
What if click event of button raises exception due to some dialog data posted on server, when it will be enabled again? I guess you should enable it always once click callback is finished?
Also test cases are missing.

Reply:
This dialog will destroy after click event, so should not enabled again.
And add test cases.